demos/serverless/src/handlers.js

```diff
@@ -9,18 +9,27 @@ const { metricScope } = require('aws-embedded-metrics');
 // Store meetings in a DynamoDB table so attendees can join by meeting title
 const ddb = new AWS.DynamoDB();
 
-// Create an AWS SDK Chime object. Region 'us-east-1' is currently required.
+// Set the AWS SDK Chime endpoint. The Chime endpoint is https://service.chime.aws.amazon.com.
+const endpoint = process.env.CHIME_ENDPOINT;
+const currentRegion = process.env.REGION;
+const useRegional = process.env.USE_REGIONAL;
+
+// Create an AWS SDK Chime object.
 // Use the MediaRegion property below in CreateMeeting to select the region
 // the meeting is hosted in.
 const chime = new AWS.Chime({ region: 'us-east-1' });
 
 // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com.
-chime.endpoint = new AWS.Endpoint(process.env.CHIME_ENDPOINT);
+chime.endpoint = new AWS.Endpoint(endpoint);
 
-const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
+const chimeRegional = new AWS.ChimeSDKMeetings({ region: currentRegion });
+chimeRegional.endpoint = new AWS.Endpoint(endpoint);
```

Comment:
You should only set endpoint if it is provided

Reply:
endpoint is set to https://service.chime.aws.amazon.com by default

so I have added the below condition so it will set the Chime Meetings SDK endpoint only if the endpoint is not equal to  https://service.chime.aws.amazon.com

if(endpoint != 'https://service.chime.aws.amazon.com'){
  chimeSDKMeetings.endpoint = new AWS.Endpoint(endpoint);
}